package.json

```diff
@@ -49,20 +47,21 @@
     "lint-staged": "^15.2.10"
   },
   "resolutions": {
-    "**/nomnom/underscore": "^1.12.1",
-    "**/core/ember-inline-svg/**/nth-check": "^2.0.1",
-    "**/ember-cli/**/ansi-html": "^0.0.9",
-    "**/ember-cli/testem/fireworm/async": "^2.6.4",
-    "**/@hashicorp/design-system-components/ember-stargate": "^0.6.0",
-    "micromatch": "^4.0.8"
+    "nomnom/underscore": "^1.12.1",
+    "@hashicorp/design-system-components/ember-stargate": "^0.6.0",
+    "micromatch": "^4.0.8",
+    "css-select/nth-check": "^2.0.1",
+    "node-gyp": "^10.0.0",
+    "ansi-html": "^0.0.9",
```

Comment:
None blocking comment: it is possible to keep `ansi-html` relative?

Reply:
@calcaide  I originally had it under ember-cli, but ansi-html is not a direct dependency and they are used in broccoli-middleware and broccoli as pointed out by Lis [here](https://github.com/hashicorp/boundary-ui/pull/2697#discussion_r1972260084). Do you think we should use two relative paths for this dependency since they are used by both broccoli and broccoli-middleware? I'm open to changing this if that option fits us better than not specifying the relative path at all.

Reply:
Ahh gotcha.

I am fine leaving it like this, because with the ember upgrade I am sure we can review the `ansi-html` resolution 😉